[2024-10-04T21:10:34.428+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-04T21:10:34.453+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_pipeline.youtube_extration manual__2024-10-04T21:10:32.567552+00:00 [queued]>
[2024-10-04T21:10:34.471+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_pipeline.youtube_extration manual__2024-10-04T21:10:32.567552+00:00 [queued]>
[2024-10-04T21:10:34.475+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-04T21:10:34.495+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): youtube_extration> on 2024-10-04 21:10:32.567552+00:00
[2024-10-04T21:10:34.502+0000] {standard_task_runner.py:72} INFO - Started process 98 to run task
[2024-10-04T21:10:34.507+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_pipeline', 'youtube_extration', 'manual__2024-10-04T21:10:32.567552+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpuo2b6la6']
[2024-10-04T21:10:34.511+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask youtube_extration
[2024-10-04T21:10:34.575+0000] {task_command.py:467} INFO - Running <TaskInstance: reddit_pipeline.youtube_extration manual__2024-10-04T21:10:32.567552+00:00 [running]> on host 85380cf6bf50
[2024-10-04T21:10:34.658+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Haziq' AIRFLOW_CTX_DAG_ID='reddit_pipeline' AIRFLOW_CTX_TASK_ID='youtube_extration' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T21:10:32.567552+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T21:10:32.567552+00:00'
[2024-10-04T21:10:34.662+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-04T21:10:34.677+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
TypeError: reddit_pipeline() missing 2 required positional arguments: 'file_name' and 'subreddit'
[2024-10-04T21:10:34.689+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=reddit_pipeline, task_id=youtube_extration, run_id=manual__2024-10-04T21:10:32.567552+00:00, execution_date=20241004T211032, start_date=20241004T211034, end_date=20241004T211034
[2024-10-04T21:10:34.705+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-04T21:10:34.708+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 11 for task youtube_extration (reddit_pipeline() missing 2 required positional arguments: 'file_name' and 'subreddit'; 98)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
TypeError: reddit_pipeline() missing 2 required positional arguments: 'file_name' and 'subreddit'
[2024-10-04T21:10:34.723+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-04T21:10:34.751+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-04T21:10:34.758+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
